monai/bundle/workflows.py

```diff
@@ -129,6 +130,23 @@ def get_workflow_type(self):
         """
         return self.workflow
 
+    def add_property(self, name: str, required: str, desc: str | None = None):
+        """
+        Besides the default predefined properties, some 3rd party aplications may need the bundle
+        dfinition to provide additonal properties for the specific use cases, if the bundlle can't
+        provide the property, means it can't work with the application.
+        This utility adds the property for the application requirements check and access.
+
+        Args:
+            name: the name of target property.
+            required: whether the property is "must-have".
+            desc: description document for the property.
```

Comment:
suggest to write like:
```
desc: descriptions for the property
```

Reply:
updated in the latest commit.

Thanks.